modules/openapi-generator/src/main/resources/typescript-axios/baseApi.mustache

```diff
@@ -20,6 +20,18 @@ export const COLLECTION_FORMATS = {
     pipes: "|",
 };
 
+/**
+ *
+ * @export
+ */
+export const JSON_MIME_PATTERN = new RegExp('application\\/json(;.*)?', 'i');
+
+/**
+ *
+ * @export
+ */
+export const JSON_VENDOR_MIME_PATTERN = new RegExp('application\\/vnd.(.*)+json(;.*)?', 'i');
```

Comment:
These should match those at https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultCodegen.java#L6606-L6607

Reply:
maybe we could create a small function that does the check, similar to 
https://github.com/OpenAPITools/openapi-generator/blob/9236d5699d57857c259f590dcbf65eac2a1a3901/modules/openapi-generator/src/main/resources/typescript-angular/configuration.mustache#L135-L148